packages/smooth_app/lib/pages/folksonomy/folksonomy_page.dart

```diff
@@ -234,11 +243,26 @@ class _FolksonomyContentState extends State<_FolksonomyContent> {
 
     if (res != null && mounted) {
       if (action == FolksonomyAction.edit) {
-        context.read<FolksonomyProvider>().editTag(res.key, res.value);
+        unawaited(
+          context.read<FolksonomyProvider>().editTag(
+            context,
+            res.key,
+            res.value,
+          ),
+        );
       } else if (action == FolksonomyAction.add) {
         try {
-          context.read<FolksonomyProvider>().addTag(res.key, res.value);
+          unawaited(
+            context.read<FolksonomyProvider>().addTag(
+              context,
+              res.key,
+              res.value,
+            ),
+          );
         } catch (e) {
+          if (!mounted) {
```

Comment:
Don't know what we're supposed to catch here, but that's OK.
It will be refactored with background tasks.